package.json

```diff
@@ -40,6 +40,7 @@
     "expo-asset": "8.2.0",
     "expo-constants": "9.2.0",
     "expo-font": "8.3.0",
+    "expo-linking": "^1.0.4",
```

Comment:
https://github.com/NMF-earth/nmf-app/blob/master/contributing.md : no ^

Reply:
Hi @PierreBresson thanks for the comment, made the change, Also fixed the carat in the prompt-sync package